### DISCUSSION TITLE
Take the 2021 Miller User Survey!

### DISCUSSION THREAD

**Discussion by johnkerl** [johnkerl/miller#542] [Announcements]
https://docs.google.com/forms/d/e/1FAIpQLSfNgLS9WVRq9mu8dZlMbS7LbTyRH1diRDnT_dGiavSOh_A8xA/viewform?usp=sf_link

It takes only a few minutes -- we would love to know how Miller can help you solve your problems!

Update: survey closed 2021-12-28.

**Comment by aborruso** [comment]
Hi @johnkerl in discussions you have the "announcements" category https://github.com/github/feedback/discussions/3979

For this kind of post I would use it. Moreover I will pin it to give it more evidence

A sample pinned post:
![image](https://user-images.githubusercontent.com/30607/120097661-1c2a6c80-c132-11eb-9c9f-e44be4301bed.png)

**Reply by johnkerl** [reply]
thanks @aborruso !

**Comment by johnkerl** [comment]
Results are in!

https://github.com/johnkerl/miller/discussions/670
